src/application/components/Layouts/SettingsModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { Dispatch, Fragment, MutableRefObject, SetStateAction } from "react";
+import { Dialog, Transition } from "@headlessui/react";
+
+declare const VERSION: string;
+
+export function Modal({
+  open,
+  setOpen,
+  cancelButtonRef,
+}: {
+  open: boolean;
+  cancelButtonRef: MutableRefObject<null>;
+  setOpen: Dispatch<SetStateAction<boolean>>;
+}) {
+  return (
+    <Transition.Root show={open} as={Fragment}>
+      <Dialog
+        as="div"
```

Comment:
```suggestion
```

[nit] Looks like `div` is the default, so feel free to omit this.

Reply:
Addressed in https://github.com/apollographql/apollo-client-devtools/pull/1128/commits/41bdf86d24adcb2decbd996401584e8c7914dc99